package.json

```diff
@@ -50,7 +50,7 @@
         "fast-text-encoding": "^1.0.3",
         "grapheme-splitter": "^1.0.4",
         "js-base64": "^3.7.2",
-        "pusher-js": "^7.0.6",
+        "pusher-js": "7.4.0",

```

Comment:
Is this specific to this PR, if yes, why `7.4.0` specifically and not the latest(or closer to the latest) one?

Reply:
This is to fix the button keyboard Buffer error problem

Reply:
It's related to this issue https://github.com/ripe-tech/ripe-robin-revamp/issues/481